gprofiler/utils/__init__.py

```diff
@@ -267,15 +282,28 @@ def run_process(
 
 
 def pgrep_exe(match: str) -> List[Process]:
-    pattern = re.compile(match)
-    procs = []
-    for process in psutil.process_iter():
-        try:
-            if pattern.match(process_exe(process)):
-                procs.append(process)
-        except psutil.NoSuchProcess:  # process might have died meanwhile
-            continue
-    return procs
+    if is_windows():
+        pythoncom.CoInitialize()
+        w = wmi.WMI()
+        return [
+            Process(pid=p.ProcessId)
+            for p in w.Win32_Process()
```

Comment:
Why using this library to iterate and not psutil?

Reply:
I noticed psutil wasn't including all python processes running on Windows. This is why I found an alternative

Reply:
Okay, let's document it in the code ("psutil doesn't include all processes ...")

Reply:
If the implementation is completely separate, it is more readable to read the code if structured like this:
```
if is_windows():
    def pgrep_exe(...):
          windows one
else:
    def pgrep_exe(...):
          linux one
```

Reply:
Done